textdb/textdb-common/src/main/java/edu/uci/ics/textdb/common/utils/Utils.java

```diff
@@ -78,10 +79,20 @@ public static IndexableField getLuceneField(FieldType fieldType,
                 luceneField = new org.apache.lucene.document.StringField(fieldName, dateString, Store.YES);
                 break;
             case TEXT:
-	            luceneField = new org.apache.lucene.document.TextField(
-	                    fieldName, (String) fieldValue, Store.YES);
-	            break;
-            
+                org.apache.lucene.document.FieldType luceneFieldType = new org.apache.lucene.document.FieldType();

```

Comment:
Add comments to the codebase: "By default we enable positional indexing in Lucene so that we can return information about character offsets and token offsets.""